### PR TITLE
expire improvements

### PR DESCRIPTION
**Pull request by awick**
* Handle 'G' at the end of freeSpaceG
* improve expire logging
* Actually enforce that we keep 10 files around

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: viewer/viewer.js [hunk 1]
```diff
@@ -927,7 +927,7 @@ function expireDevice (nodes, dirs, minFreeSpaceG, nextCb) {
   }
   const query = {
     _source: ['num', 'name', 'first', 'size', 'node', 'indexFilename'],
-    from: '0',
```
Comment (vpiserchia) [inline viewer/viewer.js:930]: +1

File: viewer/viewer.js [hunk 2]
```diff
@@ -971,9 +971,7 @@ function expireDevice (nodes, dirs, minFreeSpaceG, nextCb) {
     }
 
     if (data.hits.total <= 10) {
```
Comment (vpiserchia) [inline viewer/viewer.js:973]: I think that this check is wrong now, it should be:
```
if (data.hits.total == 0) {
```

Comment (vpiserchia) [reply viewer/viewer.js:973]: I checked again and everything works fine, sorry for the noise.

Comment (awick) [reply viewer/viewer.js:973]: np - hits.total is the number of matching documents ignoring pagination (from)

**Pull request merged**